frame/balances/src/lib.rs

```diff
@@ -213,7 +213,41 @@ pub mod pallet {
 
 	pub type CreditOf<T, I> = Credit<<T as frame_system::Config>::AccountId, Pallet<T, I>>;
 
-	#[pallet::config]
+	pub mod config_preludes {
+		use super::*;
+
+		pub struct TestDefaultConfig;
+		#[frame_support::register_default_impl(TestDefaultConfig)]
+		impl DefaultConfig for TestDefaultConfig {
+			type Balance = u64;
```

Comment:
Is there any advantage for this being u64 when most production chains will be u128?

Reply:
This is probably just copy paste since most test configs still use `u64`. A minor advantage could that this makes it harder to silently depend on the type to be `u128` if we use a different one in tests.